index.js

```diff
@@ -3,6 +3,8 @@ var RSS = require('rss');
 var express = require('express');
 var app = express();
 
+var regex_exclude = process.env.DOCKER_HUB_RSS_REGEX_EXCLUDE || false;
```

Comment:
My fault, I'm always on the fence about which branch should be the default. I'll release the dev branch now so you can get the latest and greatest!